pkg/download/classic/downloader.go

```diff
@@ -49,83 +49,37 @@ type Downloader struct {
 	client dtclient.Client
 }
 
-// WithAPIContentFilters sets the api content filters for the Downloader - these will be used in addition to base API filtering
-func WithAPIContentFilters(apiFilters map[string]contentFilter) func(*Downloader) {
-	return func(d *Downloader) {
-		d.apiContentFilters = apiFilters
-	}
-}
-
-func WithAPIs(apis api.APIs) func(*Downloader) {
```

Comment:
nit: Technically WithAPIs and the new WithEndpoints have the same meaning, they set d.apisToDownload.
Why the rename?